stylesheets/style2.css

```
@@ -0,0 +1,436 @@
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', serif;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* ---------------------- 3.1 Reusables ----------------------- */
+
+.outerContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.innerContainer {
+    flex-basis: 1200px;
+    display: flex;
+    flex-direction: column;
+    margin: 0 1rem;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 1rem;
+}
+/* ---------------------- 3.2 Specific ------------------------ */
+/* ------------------------------- 4. Areas ---------------------------------------- */
+/* ---------------------- 4.1 Header -------------------------- */
+header, #contactUs {
+    background: linear-gradient(320deg, rgba(87,34,205,1) 18%, rgba(72,69,228,1) 44%);
+    color: white;
+    position: relative;
+    isolation: isolate;
+}
+
+/*------Bonus : Texture noise on background-------*/
+header::after,
+#contactUs::after {
+    content: '';
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    z-index: -1;
+    background-image: url("../assests/noise-texture.png");
+    opacity: 0.9;
+}
+
+nav {
+    display: flex;
+    gap: 2rem;
+    justify-content: flex-end;
+    padding-top: 1rem;
+    margin-right: 3rem;
+}
+
+nav > a {
+    text-decoration: none;
+    color: white;
+}
+
+header > section {
+    position: relative;
+    padding-top: 3rem;
+    padding-bottom: 7rem;
+}
+
+header > section > div > h1 {
+    font-size: 2rem;
+}
+
+.workTogether {
+    background-color: var(--pink-color);
+    font-size: 0.7em;
+    padding: 15px;
+    margin-top: 1em;
+    border-radius: 35px;
+    width: 10rem;
+    border: none;
+    color: white;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.50);
+    text-transform: uppercase;
+    align-self: center;
+    cursor: pointer;
+}
+
+.workTogether:active {
+    background-color: #fc7bff;
+}
+
+.image-squiggle-stripe {
+    background-image: url(../assests/squiggle_line_stripe.svg);
+    position: absolute;
+    width: 79px;             /* REM Units*/
+    height: 80px;            /* REM Units*/
+    bottom: 0;
+    left: calc(50% - 40px);
+}
+
+header > section > .innerContainer::before {
+    content: '';
+    border-style: solid;
+    border-width: 50px 98vw 33px 0; /* adjust the border-width values to change the size of the triangle */
+    border-color: transparent transparent white transparent; /*color*/
+    position: absolute;
+    bottom: 0;
+    left: 0;
+}
+/* --------------- 4.1.1 Navigation Bar ---------------- */
+/* --------------- 4.1.2 Hero Section ------------------ */
+/* ---------------------- 4.2 Main ---------------------------- */
+/* --------------- 4.2.1 Introduction ------------------ */
+.introduction {
+    color: var(--blue-color);
+    padding: 3em 0 3em 0;
```

Dit zou je korter kunnen schrijven
```suggestion
    padding: 3em 0;
```